src/lib/blueprint.ts

```diff
@@ -840,6 +846,20 @@ const createResource = (
   }
 }
 
+const validateRoutePath = (
+  routePath: string | undefined,
+  routes: Route[],
+): string => {
+  if (routePath == null || routePath.length === 0) {
+    throw new Error('Missing route path')
```

Comment:
Let's make sure this error can include the resource name that is missing the route path

Reply:
551961160de9c9afc246911b4ab2a43e96ea6c0e